routes/article.js

```diff
@@ -138,29 +136,54 @@ router.post('/vote', (req, res) => {
             })
             console.log(`article: ${articleChanged}`)
             res.send(articleChanged)
-            return
-
         })
-        .catch((error) => {
+        .catch((err) => {
             res.status(500).send({
-                error,
+                err,
                 reqBody: req.body
             })
-            return
         })
 })
 
-router.get('/', (req, res) => {
-    Article
-        .find()
-        .lean()
-        .then((articles) => {
-            res.send(articles) 
+router.get('/top', (req, res) => {
+    let findParams = {}
+    if (req.query.lastVote && req.query.lastDate) {
+        findParams = {
+            votes: { $lte: req.query.lastVote },
+            publishedAt: { $lt: req.query.lastDate }
+        }
+    }
+
+    const sortParams = {
+        votes: -1,
+        publishedAt: -1
```

Comment:
I couldn't find in the docs what setting values to -1 does, can you link it or explain it pls?